include/gauge_field_order.h

```diff
@@ -56,8 +58,8 @@ namespace quda {
 	 @param[in] x_cb Checkerboarded space-time index we are accessing
 	 @param[in] parity Parity we are accessing
        */
-      __device__ __host__ inline gauge_wrapper<Float,T>(T &gauge, int dim, int x_cb, int parity)
-	: gauge(gauge), dim(dim), x_cb(x_cb), parity(parity) { }
+      __device__ __host__ inline gauge_wrapper<Float,T>(T &gauge, int dim, int x_cb, int parity, Float phase=10.0)
```

Comment:
any reason why 10.0 is the default here?

Reply:
debugging artifact. I think I caught all places where I used that now.